plugins/coverimage.koplugin/main.lua

```diff
@@ -581,7 +581,7 @@ end
 
 function CoverImage:menu_entry_background(color)
     return {
-        text = T(_("Fit to screen, %1 background"), color),
+        text = T(_("Fit to screen, %1 background"), _(color)),
```

Comment:
That won't work in that form, but you shouldn't do this regardless. Use three strings written out in full.

Reply:
Also CoverImage:menu_entry_cache() doesn't abide by the style guide, should be menuEntryCache

Reply:
That is, the call should look like this (except it shouldn't because it's being clever and Anglocentric).

self:menuEntryBackground(_("gray"))